### ISSUE TITLE
3.3.1  Student- Accessibility Fixes - Level A -Error Identification / Remediation 

### ISSUE THREAD

**Issue by michesmith** [bcgov/SIMS#3108]
**User Story**
**As a student, I need to know why errors are caused and how to remediate them when filling out forms 

**Context:**
- These issues were uncovered through accessibility audit via CITZ. 
- Detailed Spreadsheet for Student s inserted inline and also attached here: [2024_Student Application_Audit.xlsx](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBN0xsQXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--578821d4addd12c1babcafba5c861e1f5456bc9c/2024_Student%20Application_Audit.xlsx)
- If an input error is automatically detected, the item that is in error is identified and the error is described to the user in text.

**The Problem**
"Start Application" caused an unknown error, but the cause of the error was not clear.

**Student Pages** 
Adjust the following Accessibility Criteria for the following pages:
- [ ] Profile
- [ ] Start new application 
- [ ] My applications 

**Acceptance Criteria**
- [ ] Identify areas that trigger an error
- [ ] All errors should have a cause that is clearly defined, or at least provide instructions on steps to take for remediation.

**Comment by michesmith** [comment]
Follow up with Rowan on Expectation on triggers

**Comment by michesmith** [comment]
We've this in another ticket as it was a bug.  Content for errors is being reviewed and will be captured during that process.
